packages/aws-rfdk/lib/core/lib/mountable-efs.ts

```diff
@@ -75,6 +98,23 @@ export class MountableEfs implements IMountableLinuxFilesystem {
       throw new Error('Target instance must be Linux.');
     }
 
+    target.node.addDependency(this.props.filesystem.mountTargetsAvailable);
```

Comment:
Does this work if it's `target.node?.` ?  Or, alternatively, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty ?

Then we don't have to change the definition of `IMountingInstance`

Reply:
I think `MountableEfs` already requires a construct, unless I'm mistaken. It needs to be able to modify the `IMountingInstance`'s user data, right? On that basis, there *should* be no harm in this addition. Perhaps I'm missing some corner case though...

Reply:
I remember there was a reason that I didn't put `IConstruct` in the original definition. I don't recall what that reason was... some corner case.

Reply:
Looks like I should be able to use [`Construct.isConstruct(...)`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Construct.html#static-iswbrconstructx). Changing to that....